core/src/test/scala/integration/kafka/api/ConsumerBounceTest.scala

```diff
@@ -188,14 +200,14 @@ class ConsumerBounceTest extends BaseRequestTest with Logging {
     }
 
     sendRecords(numRecords, newtopic)
-    receiveRecords(consumer, numRecords, newtopic, 10000)
+    receiveRecords(consumer, numRecords, 10000)
```

Comment:
`receiveRecords` used to accept a `topic` parameter which it did not use at all, so I removed it

Reply:
If you're asking in a more general sense - I added a way to handle exceptions in `receiveRecords` through a callback. This was to ease implementation since I needed to run `receiveRecords()` in parallel. I played a lot with these tests, now I clearly see that callback logic isn't needed.